keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -296,55 +335,93 @@ _This section must be completed when targeting beta graduation to a release._
   taint.
   Since use of this feature requires applying a taint manually by the user,
   it should not specifically require rollback.
 
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
   We will manually test upgrade from 1.25 to 1.26 and rollback from 1.26 to 1.25.
 
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
-fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
   No.
 
 ### Monitoring Requirements
 
-_This section must be completed when targeting beta graduation to a release._
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
 
-* **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
   An operator, the person who operates the cluster, can check if the
   `NodeOutOfServiceVolumeDetach` feature gate is enabled and if there is an
   `out-of-service` taint on the shutdown node.
   The usage of this feature requires the manual step of applying a taint
   so the operator should be the one applying it.
 
-* **What are the SLIs (Service Level Indicators) an operator can use to determine
-the health of the service?**
-  - [ ] Metrics
-    - Metric name: We can add new metrics deleting_pods_total, deleting_pods_error_total
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+  - [X] Metrics
+    - Metric name: 
+      - We can add new metrics `deleting_pods_total`, deleting_pods_error_total
```

Comment:
Can you add `` to deleting_pods_error_total as well?

Reply:
Thank you. done.